source/slang/slang-parser.cpp

```diff
@@ -4790,6 +4790,10 @@ namespace Slang
         case TokenType::OpMod:
             return Precedence::Multiplicative;
         default:
+            if (token.getContent() == "is" || token.getContent() == "as")
```

Comment:
Seems a little unfortunate - but I guess the idea is not wanting to make 'is' or 'as' TokenType?

Reply:
Yeah, we probably want to allow using these as identifier names.